transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -173,9 +173,17 @@ private Metacard createMetacard(InputStream input, Metadata metadata, String uri
             metacard.setAttribute(new AttributeImpl(Metacard.METADATA, metacardMetadata));
         }
 
-        String lat = metadata.get(Metadata.LATITUDE);
-        String lon = metadata.get(Metadata.LONGITUDE);
-        String wkt = toWkt(lon, lat);
+		String coordinates = metadata.get("Corner Coordinates:");
+		String wkt = null;
+		if (StringUtils.isNotBlank(coordinates)) {
+			String[] lowerLeftCoordinates = getCoordinates(coordinates, "Lower Left").split(",");       
+			String[] upperRightCoordinates = getCoordinates(coordinates, "Upper Right").split(",");
+			wkt = toWkt(lowerLeftCoordinates[1], lowerLeftCoordinates[0], upperRightCoordinates[1], upperRightCoordinates[0]); 
```

Comment:
The metacard location expects WKT to be in WGS84 but it does not look like the coordinates given by gdal used by Tika is not guaranteed to come back in WGS84 depending on the input file.  Did you try this with a files that use different coordinate systems?

http://www.gdal.org/gdalinfo.html